Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -127,21 +129,41 @@ private extension ReceiptRenderer {
         for line in lines {
             summaryContent += "<tr><td>\(line.title) × \(line.quantity)</td><td>\(line.amount) \(parameters.currency.uppercased())</td></tr>"
         }
-        summaryContent += """
-                            <tr>
-                                <td>
-                                    \(Localization.amountPaidSectionTitle)
-                                </td>
-                                <td>
-                                    \(formattedAmount()) \(parameters.currency.uppercased())
-                                </td>
-                            </tr>
-                            """
+        summaryContent += taxesSummaryRow()
+        summaryContent += amountPaidSummaryRow()
         summaryContent += "</table>"
 
         return summaryContent
     }
 
+    private func taxesSummaryRow() -> String {
+        return taxSummaryRow(title: Localization.totalTaxLineDescription, amount: cartTotals.totalTax)
+    }
+
+    private func taxSummaryRow(title: String, amount: String) -> String {
+        guard amount != "0.00" else {
```

Comment:
This will work for now, although comparisons to floats make me nervous and some currencies are non-decimal.

Could we at least try amount < 0.01 instead for now?

Reply:
Hmm. I'd come to this conclusion because it's a String all the way from the API response. I didn't really want to do a conversion to a number in order to make this comparison, but I know what you mean about non-decimal currencies.

I'll have a noodle with turning it into an `NSDecimalNumber` to do a comparison, and see how it goes.